src/network/network.cpp

```diff
@@ -1448,10 +1448,9 @@ void Network::Server_Client_Joined(const char* name, const std::string &keyhash,
 		const char * player_name = (const char *) player->name.c_str();
 		format_string(text, 256, STR_MULTIPLAYER_PLAYER_HAS_JOINED_THE_GAME, &player_name);
 		chat_history_add(text);
-		Server_Send_MAP(&connection);
-		gNetwork.Server_Send_EVENT_PLAYER_JOINED(player_name);
-		Server_Send_GROUPLIST(connection);
-		Server_Send_PLAYERLIST();
+		rct_object_entry object_entries[OBJECT_ENTRY_COUNT];
```

Comment:
I know, do you have any suggestions? I can move it to heap or change it to use `std::vector` like in other cases. I don't quite expect parks to use exclusively CSO.